src/meta/proto-conv/tests/it/v032_file_format_params.rs

```diff
@@ -48,6 +48,7 @@ fn test_decode_v32_csv_file_format_params() -> anyhow::Result<()> {
             nan_display: "nan".to_string(),
             escape: "\\".to_string(),
             quote: "\'".to_string(),
+            error_on_column_count_mismatch: true,
```

Comment:
It would be better to use `false` as the default value for easier understanding of our options. What do you think about using `disable_column_count_check` or something similar?

Reply:
This PR we keep the same option and behavior as snowflake.